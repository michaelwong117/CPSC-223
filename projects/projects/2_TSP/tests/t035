#!/bin/bash
# 512 cities, farthest

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./TSP /c/cs223/hw2/Tests/t035.in -farthest < /dev/null
