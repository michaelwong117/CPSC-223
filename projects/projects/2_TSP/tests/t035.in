512
LEI UEO BGN SJP RFD NLA AUM KLL GNA UST BHI MS1 05A UOX DUR AMP OAJ BYU KKI BXE PCG WBR CXY AFA IZO PWT LBQ SUU TKK LTO AMH MBK LAS OOM TDZ MCI CBF IFL CIH ENV FDY WA7 HAB KGS SQV MLD TGM TIX SMN HZL FOC CGX MBZ VLL HGZ PHN PMI PCD INT LKV PLZ YUM SCM ERD HZH NCL LRL KIV RRG TJM GDL CA4 PAQ ULN TIE ILZ KLU BSG YOP JMN MUT VGC ME3 CMG MMH BID BMA YXZ KUA UKB SCQ LUG CXH VWT GGN NBC TUC DWA SXL RAC MOG TCQ GOH FRD CAH SIK MLT BMJ VST PTK GRP GFA GRA KSC CCC EVM BHX VWW ZXE YOD APC IDI NKI AXV THV ARE SXM BXU AZ3 PHI FRF BOY K01 SZF DIG KHV GUF OCE CPT OHD MGC AVX IAD PQQ FQJ MS3 HDF OSU PGM RDA JEF 1G4 RBF BKK BIR LAR FKB BYK HSV PUB CFR USH ZPB ELA CGC PGH A56 KBS YZH CAP PCE CZE XRH DTR CPO A69 A61 GAU PET WFK PTR ORI LHW PDS DEL MWL BGI BCB CHT OSP URG KWJ TVC ILL SKW ITM TAL MMQ ZCO DIV MTD DXR A03 BBS MZG MSH DUB CNE LOI HND SLS LUE BRY TOD CEG MJV UDD GLN A87 CIA VQS NKX NAP YSP THA OMO DUD WIC BJU 04A PNR CCY RAE MWD ARM MSP FYV SNR BBF UXM BVB LNP SBW ZBX HPT CJC AK3 SAV HUH LAK TFF WSZ A94 MIC RSA YGX JFN A53 A07 LAO QAX VWF YOG EHT NY8 ATC ILQ WSJ WMO A50 ERV BVI YCK TEZ YHZ NIN MEN OGA PEN PPP POC SLN KTT OQX CLC HSI WHO JAL BVR WUX FZO TBN MYP DPA ADQ MCR XWN MSA CT2 DND LXV MYW TIF LGP XMR TCC B21 YVN PIM FIC WQD DGT LFO PNA PTN MI4 A72 YDU EKN YVP YQQ DBV FPY NSY A93 DQK MFA MIQ KNU DYS AUG BQV AK6 VOG PKU NCP VWL HAY LZR RNB NKT MKE PKE GNB AF4 FDE OGN WTT PSD AHS JGC SML KWF FQA DQD JXN FMY DKR NUP YKU ACA WAS AKC SGP SFG BKS S24 STQ OXR EG1 MQY FNI VZL QBY YNT KTA ABY FJR HDS YXJ PMP KGG CED MGF CCK MKK YAF ALM AUS BEV AKB TIZ MMK MHG EAG KEN JZR CAJ JKT RIC TX3 OUE FVW MQP MOU YPR MUL VIG NAO TYN NHV YGN MDY MGJ DRW HRO NET RHI CBB ANO REP MRK YAT SVA A28 UXP NCH THR TGR NSF UII BIK TNU GLF SNS OZP MI3 BHZ SAT BFF AYS CZM YQZ LDV BOD GIZ PCA FDU BZR LLI BOR MCD BAK BSM ALB WRI YGW CRX UXA HYD RYE ROB TX7 DEN REY WNP SHD WSP SFD OBF OSL TFN YGJ IWO ION TOV BCI SYX EED CCS VZA ONP MOZ OHR OH1 LAX YLQ DEH PGA NC2 CUP CEO PIB NGE TX9 BNB CAR AF3 BHE A71 BAZ OEO COO DIK WKN EUF NVT FSP IXW CIG ZLT OTU
36.843889 -2.370000
26.363611 126.713889
16.621111 121.250278
-20.817222 -49.406944
42.195278 -89.097222
-12.998333 28.665000
43.662778 -92.933056
59.128056 -156.858611
53.601944 24.053889
29.959167 -81.339722
-38.725556 -62.168889
33.832500 -89.798056
67.570000 -148.183889
34.384444 -89.536944
-29.611667 31.119444
-24.699444 44.732778
34.829167 -77.612222
49.984722 11.640000
60.913889 -161.493333
14.847222 -12.468333
17.261111 -90.246111
43.722778 -85.504167
25.554444 -79.275278
-34.587778 -68.403333
35.413611 132.890000
47.490278 -122.764722
-0.705556 10.244167
38.264444 -121.924167
7.461944 151.843056
25.990000 -111.348333
6.039444 37.590833
-10.233333 -54.950000
36.080000 -115.152222
-36.308333 148.973333
41.650000 -83.533333
39.297500 -94.713889
41.260000 -95.758611
-17.559444 146.011667
36.183333 113.100000
40.718611 -114.030833
41.011944 -83.668611
47.833333 -120.033333
34.129167 -87.994444
36.794722 27.091111
48.098056 -123.187222
42.170556 -112.289444
46.467778 24.412500
28.514722 -80.799167
45.123889 -113.881389
40.986667 -75.995000
25.934722 119.663611
41.858889 -87.608333
-3.372222 -57.724722
41.706389 -4.851944
66.191667 -155.668889
42.910833 -82.528889
39.551667 2.738889
43.019167 -91.123611
36.133611 -80.221944
42.161111 -120.399167
-33.985000 25.617222
32.656667 -114.605833
61.844444 -165.573611
46.733333 36.783333
26.322222 109.150000
55.038056 -1.689722
9.767222 1.091111
46.927778 28.930833
-19.757778 63.361111
57.179722 65.319444
20.521944 -103.311111
41.777500 -82.678056
61.595000 -149.088611
47.842500 106.767500
7.201111 35.415278
49.233333 18.613611
46.642500 14.337778
1.902500 9.805833
58.491389 -119.408333
44.163889 -94.004167
41.367778 -91.148333
42.843333 -75.561111
44.409444 -69.011944
-19.011944 -57.673056
37.631111 -118.851389
41.168056 -71.577778
59.354444 17.941389
47.966944 -84.786389
3.775278 103.209167
34.632778 135.223889
42.896389 -8.415278
46.004167 8.910556
49.300000 -123.133333
32.110833 -84.188889
6.103611 -5.986944
55.564722 52.092500
-26.833333 -65.200000
38.579444 -121.856944
54.280278 -8.599167
42.761111 -87.813889
20.518056 99.258333
-18.053333 -70.275833
64.190833 -51.678056
48.521944 -123.024444
9.177500 105.177778
36.898889 -89.561667
45.647778 -68.685556
7.370833 -60.489722
59.589444 16.633611
42.665556 -83.420556
-11.739722 -49.132222
47.504722 -111.187222
8.328333 -73.691944
48.663056 21.241111
22.460833 -78.328611
47.425000 -92.498056
52.453889 -1.748056
46.386667 -112.764444
52.845833 0.766111
54.405000 -110.279444
38.213056 -122.280556
40.632222 -79.105556
55.875000 -133.220833
40.493611 -84.298056
39.916944 -76.873056
18.450000 -66.675278
18.040833 -63.108889
8.951389 125.483889
33.526944 -112.295278
-2.474722 -45.105278
50.030278 8.588056
11.159722 -4.331111
36.800000 -86.810278
41.254444 36.567222
27.792222 99.678056
48.528056 135.188333
30.289722 -87.671667
38.310556 -75.123889
-33.964722 18.601667
41.180000 20.742222
41.703333 -86.821111
33.405000 -118.415833
38.947500 -77.460000
-31.435833 152.863333
58.098611 -134.760000
32.799167 -89.126111
53.878889 14.151389
40.079722 -83.073056
59.348333 -151.831667
-18.950000 135.200000
38.591111 -92.156111
35.986111 -113.816944
34.263889 -116.856111
13.680833 100.747778
26.482222 87.263611
41.311944 -105.675000
48.779444 8.080556
7.738889 -5.073889
34.637222 -86.775000
38.290000 -104.498056
49.173333 -0.450000
-54.802778 -68.327778
53.891111 -92.196389
29.600556 -96.321944
-5.452778 148.423611
29.033611 79.472222
56.036667 -160.266111
7.944444 -11.761111
55.293056 -114.777222
19.732778 -72.194722
57.153889 -157.396944
11.414722 -69.681667
-33.600556 150.780833
48.510000 -122.803056
-27.261111 -70.779167
55.740000 -162.125278
54.582500 -164.914444
26.107778 91.585833
-31.718333 -52.327778
47.285556 -68.312778
57.833333 -153.250000
57.885278 -152.846111
36.515278 103.620833
28.627500 -100.535278
28.559444 77.099444
32.781667 -98.060278
13.074722 -59.492222
37.207778 -80.407778
-43.811944 -176.465556
54.478611 17.104167
-29.782222 -57.038056
35.126389 126.808889
44.741667 -85.582222
45.116944 -95.129444
61.965278 -151.191389
34.785556 135.438056
65.174444 -152.108056
-8.858056 31.334444
-38.766944 -72.637222
5.833333 -5.366667
-16.978333 130.555000
41.371667 -73.482222
58.457500 -154.023333
51.323889 -0.847500
23.569167 119.618611
20.673333 58.891944
53.421389 -6.270000
38.428333 -105.106111
27.615556 -99.520000
35.553333 139.781111
44.055000 27.180000
48.340000 19.735556
37.814444 -85.499722
2.818056 104.160000
53.178056 -2.977778
37.778889 -0.813333
33.748333 -116.274722
29.026389 -10.050556
58.583056 -152.445833
41.799444 12.595000
18.134722 -65.493611
32.868333 -117.141667
40.884444 14.290833
48.757222 -86.345833
35.380000 -86.246667
43.282778 17.845833
-45.929167 170.197500
58.458889 -3.093056
29.503611 81.668333
68.083333 -163.166667
-4.816111 11.886667
43.072500 -92.610833
30.906667 41.138333
32.563056 71.570556
-30.528056 151.617222
44.881944 -93.221667
36.005000 -94.170000
47.312222 -2.149167
42.508333 -71.183333
61.044444 -158.145278
2.846389 -60.690000
36.987500 -82.530000
2.261944 111.985556
36.651667 -93.345833
42.723611 -93.226389
-22.498333 -68.903611
64.198056 -147.928611
32.127500 -81.202222
-16.686944 -151.021667
68.222222 -134.985833
-3.383056 -64.724167
-41.737222 171.579167
70.574167 -155.318056
45.061944 -93.353889
-36.588056 -64.275556
56.357778 -94.710833
41.778056 -80.695556
64.333333 -147.500000
66.766111 -160.152778
18.176111 120.531111
33.966667 -118.450000
43.733611 -103.619444
51.633333 -85.950000
41.753333 -72.628333
40.986944 -72.519167
24.629444 -75.673889
-17.695000 -71.343889
57.731111 -153.318333
64.689167 -163.412778
58.562500 -152.596111
29.976667 -99.085556
-25.897500 139.347500
67.021111 -126.128611
26.709722 92.784444
44.879722 -63.510278
60.020278 -151.589444
44.504167 3.527500
41.119444 -101.769722
5.297222 100.276944
-20.495000 148.552222
34.091667 -117.781667
38.790556 -97.652222
67.701111 24.846944
67.000000 -162.833333
29.556944 -95.137500
40.605278 -98.427778
-43.362222 170.132500
19.472222 -96.781944
45.692222 -85.566667
31.497500 120.430556
51.519444 -2.590833
37.741667 -92.140833
37.622500 61.896667
41.915556 -88.246667
57.750000 -152.493889
17.061111 -89.155556
26.908056 -80.328333
53.441389 -91.762778
41.932778 -72.776944
56.452500 -3.025833
39.219444 -106.316389
-10.339167 40.181667
21.483333 40.544444
13.157500 123.734722
44.498889 0.200556
35.182778 -103.603056
45.085556 -70.216389
66.591944 -61.573333
32.840556 -84.882500
55.777778 -131.537500
41.890556 -121.360556
9.334167 123.301944
5.590278 44.188889
42.770000 -1.646389
29.709444 -91.338889
44.625278 -86.200833
59.833333 -152.584722
60.291944 -102.501944
38.889444 -79.857222
58.101944 -68.406667
49.710833 -124.886667
42.561389 18.268333
30.069167 -83.580556
37.401667 14.922222
59.791667 -151.266667
44.055556 -75.720000
-7.917500 39.668611
41.196111 -96.112222
26.404444 80.414167
32.418333 -99.856667
44.320556 -69.797222
58.455556 -135.883333
64.071389 -148.000000
48.782500 44.345556
0.464167 101.438611
14.794444 120.271389
47.375278 -94.798611
65.202778 -161.152778
-14.672778 145.454167
56.267222 15.265278
34.900833 -76.880833
42.946944 -87.896944
-33.131389 148.239167
45.363056 5.329444
33.392222 62.261111
61.459167 5.847500
24.467500 122.979722
60.777222 -148.721667
31.279444 32.240000
15.472778 -84.353056
35.966667 -112.133333
23.583333 -75.268056
55.297222 -133.240556
55.606111 -131.483333
42.648611 -111.581944
42.260556 -84.460556
26.586667 -81.863333
14.740278 -17.490278
60.906111 -162.440556
53.821111 -79.005000
16.756944 -99.753889
38.883333 -77.033333
41.037500 -81.466944
-20.424444 120.140000
18.100000 -63.047222
-3.863889 102.339167
41.295833 -83.037222
41.412500 -78.502500
34.200833 -119.207222
30.116389 30.915556
36.008889 -86.520000
43.757500 4.416389
58.733333 -156.950000
41.718056 -82.821111
37.397222 121.370000
-20.712222 116.773333
31.535556 -84.194444
25.112222 56.324167
-24.347500 30.946111
56.238333 -120.740278
-6.498611 143.510556
12.572500 -12.220278
-32.130556 133.709722
-23.476389 -52.016389
-12.188611 96.830556
21.152778 -157.096389
45.791667 -71.991111
32.841389 -105.988889
30.298056 -97.701389
31.286667 34.723056
52.220556 -174.206111
-5.847222 142.947500
68.781667 32.751111
49.472778 8.513333
67.562222 -162.975278
7.896667 -11.173611
45.704444 -108.759444
6.231944 -62.854722
-6.153056 106.848889
37.505278 -77.319722
26.166111 -97.345833
1.616389 16.037778
65.283333 -153.000000
-25.383056 31.105556
62.094722 -163.682778
54.285833 -130.444722
31.137778 -83.704167
8.624167 -71.672778
30.800000 106.066667
37.745833 112.630556
-8.783333 -140.166667
50.850000 -126.850000
28.204444 -177.378611
41.510000 -74.264722
-12.414722 130.876944
36.261389 -93.154722
24.315278 -75.454167
45.630833 -89.466667
-17.421111 -66.176944
-16.182500 39.945000
13.410833 103.812222
25.995000 -81.672500
52.927500 -82.431944
63.686389 -170.493056
57.253889 -154.422222
30.367778 -89.454722
-10.358333 38.780278
35.689444 51.313333
33.058333 6.084722
38.810833 -76.866944
16.086667 -86.891111
-1.186667 136.085556
41.674444 -93.021667
8.653889 -83.181944
36.662778 -121.606389
37.175000 -5.616111
42.629444 -83.982222
-19.916667 -43.933333
29.533889 -98.469167
41.874167 -103.595556
31.248333 -82.395278
20.522500 -86.925556
53.026111 -122.510278
48.530278 -4.151667
44.828611 -0.715278
16.866667 42.583333
58.906389 -157.711111
-3.311389 17.381667
43.323333 3.354722
11.975000 38.980000
47.656389 7.011667
45.865000 -84.637222
40.467500 50.046667
30.194722 -97.676667
42.749167 -73.801944
40.015556 -74.591667
55.281944 -77.765278
34.915000 -88.603611
68.166667 -163.200000
17.453056 78.468889
40.426389 -104.630000
6.233889 -10.362222
33.203333 -101.338333
39.861667 -104.673056
-14.303611 -67.353611
13.585000 123.270556
38.263889 -78.896389
14.740556 -83.968889
7.883333 -67.443889
48.081667 11.283333
60.193889 11.100278
28.482778 -16.341667
35.493333 133.239167
24.784167 141.322778
1.589444 18.046944
18.450000 -64.583333
-23.565278 145.306667
18.302778 109.412222
34.766389 -114.623333
10.603056 -66.990556
60.906111 -159.639167
44.580278 -124.058056
-17.483333 -149.766667
54.684444 8.528056
39.570278 -83.420556
33.942500 -118.408056
47.409722 -72.788889
43.275556 -91.739444
36.926111 -111.448333
35.540833 -78.390278
10.660000 -63.261667
-11.426389 15.101389
31.467222 -89.336944
7.356944 13.559167
31.947222 -99.985833
-0.286667 20.883333
46.871667 -68.018056
31.091944 64.066944
-41.517778 173.869444
59.738333 -151.030000
-0.981111 -62.918333
45.309444 -92.690000
6.357222 2.384444
46.797222 -102.801944
-5.860278 155.222222
31.951389 -85.128889
-26.880000 -48.651389
46.763056 -56.174167
22.813611 86.168333
40.495278 -107.521667
50.830833 -58.975556
7.000000 -74.716667
