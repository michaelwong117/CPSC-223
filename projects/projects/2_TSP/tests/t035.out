-farthest         :    575923.43 LEI VLL SCQ PNA FNI LDV TFN GLN DKR KBS PHI GRP MBK MBZ CMG CBB TCQ ILQ REY MGF NVT URG PET BHI RSA AFA ZCO CPO CJC TUC SJP BHZ TFF BAZ OTU GRA WSP GLF VIG SFD TOV SFG BGI BMJ SXM FSP OZP MJV BXE KGG KEN ROB BVB CAJ CZE CCS VQS CUP GGN CPT HDS AMP RRG ANO LLI TIF TIE DUR MQP PLZ NLA MYW NCH MFA MMQ LFO HYD IXW ULN MOG GAU BIR TEZ KNU DIG NAO BJU PGH MSH AMH CEO PNR FDU BNB ION GIZ FJR DEL MWD AF3 AF4 MYP OUE LBQ BSG DIV BYK BOY COO LRL NGE RAE THR BEV EG1 PSD BAK SZF KGS VOG ERD SLS TJM KHV NBC KIV OHD NSY TGR DBV TGM OMO NAP CIA PMI BZR MEN KSC LUE KLU ILZ GNA OSP OBF LUG GNB XMR BOD BOR BYU FKB MHG FRF HDF RNB MMK BMA KTT SVA VZL PCA A03 KLL A50 LAR FYV MS3 FPY TIX CEG DUB NCL DND SXL WIC UXA MOU WMO 04A EAG OQX VZA HAY A07 UXM HGZ A94 A72 NIN A93 A71 FVW SKW TAL PAQ WTT AK6 05A AK3 A53 LAK BQV KWF FQJ NKI FQA FIC YPR YCK GOH YVN YOP YQZ YXJ YZH GFA JZR YOD YDU YGX YGW YKU YLQ ALB HZL VGC DQK YAT ZPB MSA YOG YSP STQ IDI EKN JFN AKC BRY JEF TNU JMN MIC MSP OEO HPT AUM CCY DEH RHI MUT PCD RFD DPA MGC CGX MKE RAC JXN WBR MI4 TVC MI3 BVR PTK AXV TDZ FDY QBY S24 CA4 OSU OH1 PHN MCD YXZ EVM CBF ILL VWL MIQ MCI TBN K01 LNP SIK SLN HSI OGA DIK VWF ZBX HRO MQY THA THV YAF YVP WFK B21 CAR MGJ CRX UOX MS1 MLT DXR CT2 AUG EHT ME3 ZLT HAB SHD IAD HSV BCB BBF WAS WRI NY8 NSF RIC BID OCE YHZ INT PIM NC2 EUF VWT OAJ NKT ABY MUL SAV AYS UST BFF KKI PGM VWW NUP A87 SCM FDE OSL VST OHR ZXE BHX FZO BBS CFR SNR ATC XWN CXY FMY PIB RYE CXH SMN DEN WA7 ORI ADQ YGN YQQ DTR GUF UXP CIG DQD LXV PUB FRD PTR WSJ SQV PWT MLD CNE MWL ENV ONP LKV TCC PTN NET SML TX7 DYS CLC PGA TX9 AUS JGC 1G4 LAS ALM EED WQD A28 IZO YGJ UKB ITM HND PCE A56 BSM ELA ERV SAT MRK PDS AZ3 YUM MMH RBF A69 AKB UDD LOI TX3 CCC CAP ARE CZM JAL PCG MCR UII AHS USH ACA GDL DWA POC NKX LAX QAX A61 SUU APC AVX HUH MOZ NHV LTO OXR SNS MKK MDY TIZ BCI BVI PEN CAH WNP PMP TKK LZR IFL PPP XRH PKE CED RDA DRW MTD SGP KTA KUA TOD PKU CCK BKS JKT SBW IWO BGN FOC MZG HZH LHW TYN CIH YNT KWJ WUX OGN UEO NCP BKK REP SYX LAO LGP DGT BXU BIK CGC WKN ARM PQQ OOM DUD CHT WHO WSZ BHE LEI
